runConfigurations/Cromwell_server.run.xml

```diff
@@ -7,12 +7,13 @@
       <env name="CROMWELL_BUILD_CENTAUR_JDBC_URL" value="jdbc:mysql://localhost:3306/cromwell_test?allowPublicKeyRetrieval=true&amp;useSSL=false&amp;rewriteBatchedStatements=true&amp;serverTimezone=UTC&amp;useInformationSchema=true" />
       <env name="CROMWELL_BUILD_RESOURCES_DIRECTORY" value="target/ci/resources" />
       <env name="CROMWELL_BUILD_PAPI_JSON_FILE" value="target/ci/resources/cromwell-centaur-service-account.json" />
-      <env name="CROMWELL_BUILD_CENTAUR_READ_LINES_LIMIT" value="128000" />
+      <env name="CROMWELL_BUILD_CENTAUR_READ_LINES_LIMIT" value="1280000" />
```

Comment:
Yikes! That's a big increase. What are the consequences of this change?

Reply:
I needed this for the `write_fofn` shards in `lots_of_inputs_scattered.wdl` which generate very large output files. Although it probably wouldn't hurt to keep this value, I had planned to revert this along with the other changes in this file.